packages/stark-build/config/build-utils.js

```diff
@@ -90,34 +101,39 @@ function getNbbAssetsConfig() {
 }
 
 /**
- * Returns assets set in .angular-cli.json file of the project.
+ * Returns assets set in angular.json file of the project.
  */
 function getApplicationAssetsConfig() {
 	const appConfig = getAngularCliAppConfig();
 
-	if (appConfig.assets) {
-		return getCopyWebpackPluginConfig(appConfig.assets);
+	if (
+		appConfig["architect"] &&
+		appConfig["architect"]["build"] &&
```

Comment:
Again, I think we should just use the dot syntax whenever possible...

Reply:
done